lib/rules/new-line-before-expect.js

```diff
@@ -12,6 +12,8 @@ var blockRegexp = /^((f|x)?(it|describe))$/
 module.exports = function (context) {
   var suiteDepth = 0
   var lastExpectNode
+  var lineWithWhitespace
```

Comment:
can we name this var to something like: `textBeforeExpect` or something like that. Because it is not only whitespace. It can be whatever.

Also if possible can you add a test case when it is not only a whitespace, ie:

```
describe("", function() {
  it("", function(){
    var a = 1;
    var b = 2; 
    expect(1).toBe(1); expect(b).toBe(2);
  });
});
```

Reply:
No prob. Good idea on the test cases.

Reply:
What is the expected output for the test case you proposed? I can't find any other examples of two expectations on the same line.

```javascript
describe("", function() {
  it("", function(){
    var a = 1;
    var b = 2; 

    expect(1).toBe(1); expect(b).toBe(2);
  });
});
```

or

```javascript
describe("", function() {
  it("", function(){
    var a = 1;
    var b = 2; 

    expect(1).toBe(1); 
    expect(b).toBe(2);
  });
});
```

Reply:
I don't think this was thought through. Whatever makes sense to you.  I think this is rather a corner case and I just want to make sure we document the way it behaves.

Reply:
I'll go with the first option (don't break apart expectations on the same line). Although it's a bad idea to have more than one expectation on a single line enforcing that does not seem like the responsibility of this rule. It would be better handled by it's own rule - `one-expectation-per-line`, or something like that.